### PR TITLE
Cherry-pick README.md update and governance docs from master branch

### PR DESCRIPTION
**Pull request by niladrih**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+> - You can’t monitor volume usage: There are hacky workarounds to run “du” regularly, but that could prove to be a performance killer, since it could put a lot of burden on your CPU and cause your filesystem cache to fill up. Not really good for a production workload.
+> - You can’t enforce hard limits on your volume’s size: Again, you can hack your way around it, with the same caveats.
+> - You are stuck with whatever filesystem your kubelet node is offering
+> - You can’t customize your filesystem:
+
+<BR>
+### All the above issues stem from the same root cause: 
+   - hostPath/LocalPVs are simple bind-mounts from the host filesystem into the pod.
+
+### The idea behind RawFile-LocalPB
```
Comment (avishnu) [inline README.md]: LocalPV

Comment (niladrih) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,40 @@
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fopenebs%2Frawfile-localpv.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fopenebs%2Frawfile-localpv?ref=badge_shield)
 
-RawFilePV
-===
+## RawFile-LocalPV
+The RawFile-LocalPV OpenEBS Data-Engine is a simialr but more flexible, yet more complex.. derication of the LocalPV-Hostpath Data-Engine. <BR>
```
Comment (avishnu) [inline README.md]: Typo - derivation
Remove double full-stop

Comment (niladrih) [reply README.md]: Fixed this.

**Review by avishnu — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,16 +1,40 @@
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fopenebs%2Frawfile-localpv.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fopenebs%2Frawfile-localpv?ref=badge_shield)
 
-RawFilePV
-===
+## RawFile-LocalPV
+The RawFile-LocalPV OpenEBS Data-Engine is a simialr but more flexible, yet more complex derivation of the LocalPV-Hostpath Data-Engine. <BR>
```
Comment (tiagolobocastro) [inline README.md]:
```suggestion
The RawFile-LocalPV OpenEBS Data-Engine is a similar but more flexible, yet more complex derivation of the LocalPV-Hostpath Data-Engine. <BR>
```

File: README.md [hunk 4]
```diff
@@ -1,16 +1,40 @@
...
 
-Prerequisite
+### Using node-based storage
+The OpenEBS LocalPV-HostPath Data-Engine makes it pretty easy to automatically provision hostPath PVs and use them in your workloads. But, there are known limitations though:
+
+> [!IMPORTANT]
+> - You can’t monitor volume usage: There are hacky workarounds to run “du” regularly, but that could prove to be a performance killer, since it could put a lot of burden on your CPU and cause your filesystem cache to fill up. Not really good for a production workload.
+> - You can’t enforce hard limits on your volume’s size: Again, you can hack your way around it, with the same caveats.
+> - You are stuck with whatever filesystem your kubelet node is offering
+> - You can’t customize your filesystem:
```
Comment (tiagolobocastro) [inline README.md]: After `:` is there something meant to be there?

File: README.md [hunk 5]
```diff
@@ -1,16 +1,40 @@
...
+
+<BR>
+### All the above issues stem from the same root cause: 
+   - hostPath/LocalPVs are simple bind-mounts from the host filesystem into the pod.
+
+### The idea behind RawFile-LocalPV 
+To use a Filesystem based 'extant file' as the emulated block device (i.e. a soft-LUN block device), and leverage the LINUX loop device to associate that soft-LUN file as a complete flexibe block device (i.e. an emulated soft disk device). At this point you can create a PV with a fileystem on it. This allows you to...
```
Comment (tiagolobocastro) [inline README.md]: extent?

```suggestion
To use a Filesystem based 'extent file' as the emulated block device (i.e. a soft-LUN block device), and leverage the LINUX loop device to associate that soft-LUN file as a complete flexibe block device (i.e. an emulated soft disk device). At this point you can create a PV with a fileystem on it. This allows you to...
```

**Review by tiagolobocastro — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (niladrih) [reply README.md]: Dunno. Seems like a good idea to end the sentence there. Replaced the `:` with a `.`.

File: README.md (hunk 3 above)
Comment (niladrih) [reply README.md]: Using this suggesting.

File: README.md (hunk 5 above)
Comment (niladrih) [reply README.md]: Yeah, probably. Using this suggestion.

**Review by orville-wright — APPROVED** [review]
Looks good, although I believe there are more benefits to RawFile than what is mentioned. (IMHO we're ``selling short`` its value above Local-PV Hostpath).
- Maybe we can add more in the next update to the Readme

**Comment by orville-wright** [comment]
@openebs/org-maintainers 

@niladrih - I think that our Governance many exclude @edrob999 from being an `Approver` of this PR for this Repo. 
- please advise if I'm wrong in my interpretation ?
- `Special maintainer` roles and PR Approval capacity is defined and limited in our `Governance docs` and `Maintainers` file. 
   - Which constrains @edrob999 PR approval ability to the Community Repo only.
   - Please see: https://github.com/openebs/community/blob/develop/SPECIAL_MAINTAINERS
- Technically a Maintainer can invite anyone to Review any PR in any Repo... but the power to seek PR Approval for Merging needs to be used very sparingly.
   - In this case, the technical details + technical value of RawFile feels a little outside of @edrob999 technical expertise, which IMHO means seeking PR approval may be a stretch for this PR. 
- Maybe this needs to be discussed or clarified in Special Maintainers Governance.

I just want our Governance to be doing things accurately in alignment with what CNCF and TOC expect from us & the project.

**Comment by niladrih** [comment]
> @openebs/org-maintainers
> 
> @niladrih - I think that our Governance many exclude @edrob999 from being an `Approver` of this PR for this Repo.
> 
> * please advise if I'm wrong in my interpretation ?
> * `Special maintainer` roles and PR Approval capacity is defined and limited in our `Governance docs` and `Maintainers` file.
>   
>   * Which constrains @edrob999 PR approval ability to the Community Repo only.
>   * Please see: https://github.com/openebs/community/blob/develop/SPECIAL_MAINTAINERS
> * Technically a Maintainer can invite anyone to Review any PR in any Repo... but the power to seek PR Approval for Merging needs to be used very sparingly.
>   
>   * In this case, the technical details + technical value of RawFile feels a little outside of @edrob999 technical expertise, which IMHO means seeking PR approval may be a stretch for this PR.
> * Maybe this needs to be discussed or clarified in Special Maintainers Governance.
> 
> I just want our Governance to be doing things accurately in alignment with what CNCF and TOC expect from us & the project.

I think that's a good point you've brought up here. In your example, Ed's review, while welcome, would be non-binding, and could not be used to resolve any form of contention in matters related to the governance docs. However, an extra pair of eyes is always a plus.

On the README.md, the technical nature of the document may require a more nuanced approach when accepting reviews, however, Ed, et al. can voice their opinions on it and it would count as a review and holds its weight in this forum.

I think the perspective you have pointed out is of value when handling matters of governance within this project. Thank you for that.

**Pull request merged**